script/check-skip-circle-ci

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Get all changes that are not in omitted folders
+changes=( `git diff --name-only --cached ':!docs' ':!script' ':!tools'` )
+
+if [[ ${#changes[@]} -gt 0 ]]; then
+	exit
+fi
```

Comment:
I just learned of `git --exit-code` and thought of this patch. That would be cleaner still:

```sh
if git diff --cached --no-patch --exit-code ':!docs' ':!script' ':!tools'; then
    exit
fi
```